### DISCUSSION TITLE
Installation pour une version pro

### DISCUSSION THREAD

**Discussion by EmmaL9** [AmauriC/tarteaucitron.js#1209] [Q&A]
Bonjour Amauri,

J'essaie d'installer le module pro.
1- Dans un projet Drupal j'ai renseigné la balise script  dans un template twig ou je trouve la balise <head>. Est-ce bien à cet endroit que je dois l'insérer ?

2- J'ai bien la fenêtre de gestion des cookies qui apparait mais je n'ai pas la possibilité de cocher "tout accepter" ou "tout refuser".
Il y a t-il des configurations spéciales à mettre en place ?

En te remerciant pour ton retour

**Comment by AmauriC** [comment]
Bonjour Emma,

Si le bandeau s'affiche, c'est que l'installation est ok.

Le reste dépend de la configuration/installation, sans l'adresse du site, difficile d'aider.

**Reply by EmmaL9** [reply]
C'est un peu nouveau pour moi l'implémentation des cookies, ^^.

Alors en effet je pense que le soucis venait du fait que je n'avais pas intégrer de services spécifiques. J'ai par exemple essayé d'ajouter youtube et je peux gérer les cookies =).

J'ai deux autres questions en suspens:
1- Pour certains services je vois cette mention "Retirez ce marqueur HTML du site" => quel est l'intérêt de laisser le marqueur ou le retirer ?
2- J'aimerais implémenter 2 services Mapbox et Click dimenssion que je ne trouve pas dans la bibliothèque de services y-a-t-il des alternatives ?

**Reply by AmauriC** [reply]
1- Si tarteaucitron est en mode manuel et que le tag de, par exemple, GA4 est toujours sur le site, il sera chargé sans consentement et chargé une 2ème fois si l'utilisateur accepte. En mode auto il faut laisser les tags sur le site.
2- Ouvrez une issue par service avec le tag html qu'il faut installer et je verrai s'il est possible de les intégrer

**Comment by EmmaL9** [comment]
Bonjour Amaury, 

Je reviens vers toi car j'ai quelques soucis au niveau de ma phase de test. Donc j'ai bien le bandeau qui apparait pour accepter, refuser ou personnaliser les cookies. Dès lors que je fais tout accepter j'ai une deuxième fenêtre qui s'ouvre et qui me suggère les mêmes choses. Pourrait-il y avoir un soucis au niveau de la configuration ?

**Reply by AmauriC** [reply]
Possible d'avoir l'url du site? Ici ou amauri@tarteaucitron.io
